packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -126,6 +136,8 @@ export class QuoteRequestor {
 
     constructor(
         private readonly _rfqtAssetOfferings: RfqtMakerAssetOfferings,
+        private readonly _altRfqApiKey: string,
```

Comment:
Is it easier to simply pass the nnew object that contains these 2 parameters?

Reply:
agreed 👍